lib/api/manipulation.js

```diff
@@ -109,6 +110,40 @@ exports.prepend = _insert(function(dom, children, parent) {
   uniqueSplice(children, 0, 0, dom, parent);
 });
 
+exports.wrap = function(wrapper) {
+  var wrapperFn = typeof wrapper === 'function' && wrapper,
+      lastIdx = this.length - 1;
+
+  _.forEach(this, function(el, i) {
+    var parent = el.parent || el.root,
+        siblings = parent.children,
+        dom, index;
+
+    if (!parent) {
+      return;
+    }
+
+    if (wrapperFn) {
+      wrapper = wrapperFn.call(el, i);
+    }
+
+    if (typeof wrapper === 'string' && !isHtml(wrapper)) {
+      wrapper = this.parents().last().find(wrapper).clone();
+    }
+
+    dom = this._makeDomArray(wrapper, i < lastIdx).slice(0, 1);
+    index = siblings.indexOf(el);
+
+    updateDOM([el], dom[0]);
+    // The previous operation removed the current element from the `siblings`
+    // array, so the `dom` array can be inserted without removing any
+    // additional elements.
+    uniqueSplice(siblings, index, 0, dom, parent);
```

Comment:
`dom` contains only a single element (after the `.slice(0, 1)` in its initialization in L134). My proposal was to replace the slice with an access of the first element (`[0]`), which removes the access in L137, but requires a new array to be created in L141. As this is a stylistic choice, I'm fine with merging as-is. It tells a lot that this is the only thing I could find, great PR :)